### PR TITLE
⬆️ Add support for Android 12L (API 32)

### PR DESCRIPTION
**Pull request by viktor-rasevych-criteo**
JIRA: DPP-3857

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -92,6 +92,10 @@ jobs:
             - api: '31'
               abi: 'x86_64' # x86 ABI is not yet available for API 31
               tag: 'google_apis'
+
+            - api: '32'
+              abi: 'x86_64'
```
Comment (TurpIF) [inline .github/workflows/main.yml:97]: Do you use the x86_64 ABI as it is not made available by Google ? (yet?) 
Could you please check if x86 is available on API 31 ? If not then maybe we can assume that on API >= 31, then only x86_64 is available and would could move/rewrite the comment above ?

Comment (viktor-rasevych-criteo) [reply .github/workflows/main.yml:97]: Yes, x86 is not available for API 31. The same is true for API 32 and 33. I guess we can assume Google won't be releasing x86 emulators. I will remove comment then

Comment (viktor-rasevych-criteo) [reply .github/workflows/main.yml:97]: image

Comment (viktor-rasevych-criteo) [reply .github/workflows/main.yml:97]: It is from [here](https://developer.android.com/about/versions/12/get)

**Review by leonardlabat — APPROVED** [review]

**Pull request merged**